### PR TITLE
build RPMs with Packit: utilize the new per-target configuration

### PR DESCRIPTION
**Pull request by TomasTomecek**
https://github.com/packit/packit/issues/1429

### CODE REVIEW

File: .packit.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+jobs:
+- job: copr_build
+  trigger: pull_request
+  targets:
+    centos-stream-8:
+      additional_modules:
+      - foreman:el8
+      - ruby:2.7
+      - nodejs:17
```
Comment (evgeni) [inline .packit.yaml]: that's a typo in my PR, it needs to be 12 :woman_facepalming:

**Comment by evgeni** [comment]
"Submit of the build failed: module_toggle: Module name 'foreman:el8 ruby:2.7 nodejs:17' must consist of two parts separated with colon, e.g. module:stream"

Huh, is it interpreting the modules as *one* string?

File: .packit.yaml (hunk 1 above)
Comment (TomasTomecek) [reply .packit.yaml]: hehe, no worries, will update it in the next round of testing

**Comment by TomasTomecek** [comment]
> "Submit of the build failed: module_toggle: Module name 'foreman:el8 ruby:2.7 nodejs:17' must consist of two parts separated with colon, e.g. module:stream"
> 
> Huh, is it interpreting the modules as _one_ string?

it seems so, I'm really confused, opened: https://github.com/fedora-copr/copr/issues/2463

**Comment by TomasTomecek** [comment]
oh nice, it works!

```
missing groups or modules: nodejs:17
```

let me fix that :D

**Comment by evgeni** [comment]
Nice!

**Comment by TomasTomecek** [comment]
cleaned up the history and is ready for upstream

**Comment by TomasTomecek** [comment]
\o/ all in production now

**Comment by TomasTomecek** [comment]
closing then, @evgeni feel free to cherrypick or copypasta the changes :)

**Comment by evgeni** [comment]
already done and things look good, thanks a ton again!

**Pull request closed without merging**